src/store/reducers/cartReducer.jsx

```diff
@@ -0,0 +1,53 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  list: [],
+};
+
+export const cartSlice = createSlice({
+  name: 'cart',
+  initialState,
+  reducers: {
+    add: (state, action) => {
```

Comment:
not consistent naming, also very broad. add what? increaseAmount would be better, or you choice of a more descriptive name